server/src/db/migrations/20230504135943_addImageUrlColumn.cjs

```diff
@@ -7,7 +7,7 @@
  */
 exports.up = async (knex) => {
     return knex.schema.alterTable("websites", (table) => {
-        table.string("imageUrl").notNullable().defaultTo(null)
+        table.string("imageUrl").notNullable().defaultTo("")
```

Comment:
`defaultTo` should be removed because it is not setting an actual value - if we want the image to be required, then why allow for an empty string? The two methods contradict each other